libs/librepcb/common/fileio/filepath.cpp

```diff
@@ -265,10 +265,11 @@ QString FilePath::cleanFileName(const QString& userInput, CleanFileNameOptions o
     // replace spaces with underscore
     if (options.testFlag(CleanFileNameOption::ReplaceSpaces)) ret.replace(' ', '_');
     // remove all invalid characters
-    QString validChars("_\\-.0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz");
+    QString validChars(" _\\-.0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz");
```

Comment:
This change seems not really related to the commit. Also: This allows `\` but not `/`? Where is it used? This won't work with UNIX paths.